ndk-svelte-components/src/lib/user/Nip05.svelte

```diff
@@ -53,9 +53,12 @@
             user.fetchProfile()
                 .then(async () => {
                     userProfile = user!.profile;
-                    if (!userProfile?.nip05) reject;
-                    nip05Valid = await user!.validateNip05(userProfile?.nip05!);
-                    resolve(userProfile!);
+                    if (!userProfile || !userProfile?.nip05) {
+                        reject;
```

Comment:
shouldn't this be a function call? `reject()` or `reject(new Error("nip-05 not found"))`. The `catch` callback function also has the same issue.

Reply:
This code has always made me feel a little ill.... I'm going to figure out a nice/simpler way to achieve this.